packages/deployment/app.example/config.yml

```diff
@@ -1,6 +1,6 @@
 <cumulus-deployment-name>:
-  stackName: <prefix>-cumulus         # stackName must match the prefix for your IAM stack
-  stackNameNoDash: <Prefix>Cumulus
+  prefix: <prefix>-cumulus         # stackName must match the prefix for your IAM stack
```

Comment:
It seems like the inline comment here is inaccurate or at least misleading, correct? `stackName` will default to your `prefix`, but if you make it something else, what would be the impact?

Reply:
Correct. Will address with the reorganization.